tests/testthat/_snaps/init.md

```diff
@@ -0,0 +1,30 @@
+# init() prompts the user for the snapshot type
+
+    Code
+      init()
+    Output
+      This project contains a DESCRIPTION file.
+      Which files should renv use for dependency discovery in this project?
+      
+      1: Use only the DESCRIPTION file. (explicit mode)
+      2: Use all files in this project. (implicit mode)
```

Comment:
The holy grail: perfect alignment without extra spaces 😆

Reply:
I tried very hard to make it happen 😄